templates/experiments/components/experiment_chat.html

```diff
@@ -47,6 +50,9 @@
                   {% include 'experiments/components/experiment_sessions_list_tags.html' with tags=message.tags.all %}
                 </td>
               {% endif %}
+              <td scope="col" class="w-1/4 overflow-y-visible">
+                {% include 'experiments/components/experiment_sessions_list_tags.html' with tags=message.get_system_tags %}
+              </td>
```

Comment:
This should be put inside the permission check block: `{% elif perms.annotations.view_customtaggeditem %}`

Reply:
[3945761](https://github.com/dimagi/open-chat-studio/pull/440/commits/39457616235039747b54db29804b02d7062afc05)